policy.md

```diff
@@ -0,0 +1,13 @@
+# Collaboration Policy
+
+The WG manages collaboration
+
+## WoT Community Group
+
+ - CG report can be taken as feedback input
```

Comment:
Suggest to reword to:
- CG reports can serve as feedback and input to the WG and IG

Reply:
```suggestion
 - CG reports can serve as feedback and input to the WG and IG
```